### PR TITLE
 #167733822 write test for request endpoint and documentation

### PR DESCRIPTION
**Pull request by evansinho**
#### What does this PR do?
Create test and documentation for the post request endpoint
#### Description of Task to be completed?
Add test to the request test file
Write documentation for the post request endpoint
#### How should this be manually tested?
Clone the project, then open the terminal and run '''npm run test'''
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
 #167733822
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by fegzycole — APPROVED** [review]
Looks good bro!

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls && nyc report --reporter=lcov",
     "clean": "rm -rf dist && mkdir dist",
     "build": "npm run clean && npm run babel-build",
-    "heroku-postbuild": "npm run build && export NODE_ENV=production && npm start",
+    "heroku-postbuild": "npm run build && export NODE_ENV=production && npm run migration && npm start",
```
Comment (Oluwafayokemi) [inline package.json:18]: Why are we changing this here?

**Comment by Oluwafayokemi** [comment]
Why do we have so much file changes @evansinho

File: package.json (hunk 1 above)
Comment (evansinho) [reply package.json:18]: I wasn't the one that did that, I just rebased and saw that

**Review by Oluwafayokemi — DISMISSED** [review]

**Comment by codecell** [comment]
Nice one bro, but just bear in mind that it'l be updated later because @fegzycole  just informed me yesterday of new changes to the create trip request endpoint, so when i update the validations, the tests will need to be shaken too or i may just update the tests after updating the validations if it's ok by you

**Review by codecell — APPROVED** [review]

**Review by Ileriayo — APPROVED** [review]
I observed that some file changes are as a result of rebasing from develop.
If you rebase more often while you work, there will be less file changes and your PR will be easier to review.

I am guilty of this as well :). I just learnt that today.

Kindly refer to this article for more details https://github.com/andela/bestpractices/wiki/Git-Workflow#the-workflow

**Comment by evansinho** [comment]
Yeah he mentioned something like that to me, don't worry when his changes are merged, I would do something about the test.thanks

**Comment by evansinho** [comment]
Noted @Ileriayo .Thanks bro

**Pull request merged**